GameStore.API/Controllers/CartController.cs

```diff
@@ -0,0 +1,59 @@
+﻿using GameStore.Application.Interfaces;
+using GameStore.Application.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GameStore.API.Controllers;
+
+[Authorize]
+[ApiController]
+[Route("api/[controller]")]
+public class CartController : ControllerBase
+{
+    private readonly ICartService _cartService;
+
+    public CartController(ICartService cartService)
+    {
+        _cartService = cartService;
+    }
+
+    [HttpGet]
+    public async Task<ActionResult<CartModel>> GetCart()
+    {
+        var cart = await _cartService.GetCurrentCartAsync();
+
+        return Ok(cart);
+    }
+
+    [HttpPost("add")]
+    public async Task<ActionResult<CartModel>> AddToCart(CreateOrderItemModel item)
+    {
+        await _cartService.AddToCartAsync(item);
+
+        return Ok();
```

Comment:
```suggestion
        return NoContent();
```
*When adding a new resource.